docs/user/modeling.md

```diff
@@ -156,3 +156,26 @@ Through us defining the model, Nautobot will now be able to dynamically load IP
 
 !!! note
     Although `Interface.ip_addresses` is a generic relation, there is only one content type (i.e. `ipam.ipaddress`) that may be related through this relation, therefore we don't have to specific this in any way.
+
+
+## Model Querysets
+
+By default, the builtin NautobotAdapter will load all objects of a given model. However, you can provide your own queryset by overriding the `get_queryset` method. Here is an example where the adapter would only load Tenant objects whose name starts with an "s".
```

Comment:
Perhaps reword this to focus more on the word 'filtering' - we should also mention 'queryset' but if we have users that don't have in-depth enough Django knowledge this might help them find this feature better.